app/views/money_tracks/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<%=form_with model: [@group, @group.money_tracks.create], class:"input-form"  do |f|%>
+
+  <nav class="header">
+  <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+    <li>
+    <a href="/"><i class="fa fa-arrow-left"></i></a>
+    </li>

```

Comment:
- [ ] From the "Add Transaction" page, when a user clicks on the "Back" button (<), the user should navigate to the transactions page for that category. At the moment, the "Back" button (<) button takes us to the homepage. Please make this change 🙏

Reply:
@codepantha thank you for pointing out this issue, surely making this change will make my App better.

Reply:
@codepantha I found the font used in the design. I have updated my App as well.